src/main/java/pages/hero.java

```diff
@@ -21,7 +21,28 @@ public String getGalleryLinkText(WebDriver driver) throws Exception {
         return CommonFuntions.getTextOfAnyElement(driver,By.xpath("//div[@class='Gallery Hero-gallery']/button[@class='Button--underlined--dark Gallery-button']/span[@class='Button-label']"),Constant.TIME_OUT);
     }
 
-    //Locating all private page's elements
+    public String getHeroEyebrowText(WebDriver driver) throws Exception {
+        return CommonFuntions.getTextOfAnyElement(driver,By.cssSelector("span.Heading-eyebrow"),Constant.TIME_OUT);
+    }
+
+    public String getHeroTitleText(WebDriver driver) throws Exception {
+        return CommonFuntions.getTextOfAnyElement(driver,By.cssSelector("h1.Heading-title"),Constant.TIME_OUT);
```

Comment:
Im curious on why are you using an explicit locator here. Would it be better to have the element 'heroTitle' used instead?

Reply:
Because the method commonFuntions.getTextOfAnyElement() was not expecting a WebElement.   I did that change, now it expects a WebElement.    Thanks for this, now the code is much more readable.